extras/rootless/containerd-rootless-setuptool.sh

```diff
@@ -273,6 +273,51 @@ cmd_entrypoint_install_buildkit() {
 	EOT
 }
 
+# CLI subcommand: "install-buildkit-containerd"
+cmd_entrypoint_install_buildkit_containerd() {
+	init
+	if ! command -v "buildkitd" >/dev/null 2>&1; then
+		ERROR "buildkitd (https://github.com/moby/buildkit) needs to be present under \$PATH"
+		exit 1
+	fi
+	if [ ! -f "${XDG_CONFIG_HOME}/buildkit/buildkitd.toml" ]; then
+		mkdir -p "${XDG_CONFIG_HOME}/buildkit"
+		cat <<-EOF > "${XDG_CONFIG_HOME}/buildkit/buildkitd.toml"
+			[worker.oci]
+			enabled = false
+
+			[worker.containerd]
+			enabled = true
+			rootless = true
+		EOF
+	fi
+	if ! systemctl --user --no-pager status "${SYSTEMD_CONTAINERD_UNIT}" >/dev/null 2>&1; then
+		ERROR "Install containerd first (\`$ARG0 install\`)"
+		exit 1
+	fi
+	NAMESPACE="${CONTAINERD_NAMESPACE:-default}"
+	ADDRESS="${XDG_RUNTIME_DIR}/buildkit-${NAMESPACE}/buildkitd.sock"
+	ROOT="${XDG_DATA_HOME}/buildkit-${NAMESPACE}"
+	cat <<-EOT | install_systemd_unit "${NAMESPACE}-${SYSTEMD_BUILDKIT_UNIT}"
+		[Unit]
+		Description=BuildKit (Rootless)
+		PartOf=${SYSTEMD_CONTAINERD_UNIT}
+
+		[Service]
+		Environment=PATH=$BIN:/sbin:/usr/sbin:$PATH
+		ExecStart="$REALPATH0" nsenter -- buildkitd --config=${XDG_CONFIG_HOME}/buildkit/buildkitd.toml --addr="unix://${ADDRESS}" --root="${ROOT}" --containerd-worker-namespace="${NAMESPACE}" --containerd-worker-snapshotter="${CONTAINERD_SNAPSHOTTER:-overlayfs}"
+		ExecReload=/bin/kill -s HUP \$MAINPID
+		RestartSec=2
+		Restart=always
+		Type=simple
+		KillMode=mixed
+
+		[Install]
+		WantedBy=default.target
+	EOT
+	INFO "Set \`export BUILDKIT_HOST=\"unix://${ADDRESS}\"\` to use this configuration of buildkitd"
```

Comment:
When `$BUILDKIT_HOST` is unset, nerdctl (both rootless and rootful) should try to use `buildkit-<NS>/buildkitd.sock` by default and then try to fall back to `buildkit-default/buildkitd.sock`, `buildkit/buildkitd.sock`

Reply:
Also, `nerdctl build` should try to use the containerd `image` exporter when the worker UUID corresponds to the containerd UUID && the worker namespace corresponds to the current namespace

Reply:
Added this feature.